Gems/PhysX/Code/Source/Pipeline/MeshExporter.cpp

```diff
@@ -136,7 +136,8 @@ namespace PhysX
             AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
             if (serializeContext)
             {
-                serializeContext->Class<MeshExporter, AZ::SceneAPI::SceneCore::ExportingComponent>()->Version(5);
+                serializeContext->Class<MeshExporter, AZ::SceneAPI::SceneCore::ExportingComponent>()
```

Comment:
question: Should this only shift by 1 or 2 depending on if it's defined or not so we don't jump a large number of versions, won't the version number now be 37 (5 + 32)?

Reply:
Okay perfect, makes sense, thanks for the explanation